modules/ROOT/pages/configure-hardening.adoc

```diff
@@ -0,0 +1,23 @@
+= Configure Environment Hardening on Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+By design, Runtime Fabric automatically provides multiple aspects of environment hardening for your Mule installation. 
```

Comment:
"..automatically applies and enables hardening measures for your Mule applications."

Something closer to that I think makes more sense to what the user is after.

Reply:
OK - got it - thank you!